module/validate.go

```diff
@@ -30,8 +39,12 @@ func (m *Module) InitContext(ctx pgs.BuildContext) {
 func (m *Module) Name() string { return validatorName }
 
 func (m *Module) Execute(targets map[string]pgs.File, pkgs map[string]pgs.Package) []pgs.Artifact {
-	lang := m.Parameters().Str(langParam)
-	m.Assert(lang != "", "`lang` parameter must be set")
+	lang := m.lang
+	if lang == "" {
```

Comment:
You likely need to add in logic to make sure that the `lang` option is NOT set if you called `ValidatorForLanguage`

Reply:
Added a warning if the lang parameter is specified and conflicts with the lang set by ValidatorForLanguage. If you'd like, I could make it more strict and just warn on any lang parameter.

Reply:
i would error with "unknown option" if lang is set to be honest - the idea behind these binaries is there is no language option, the binaries are instead language-specific, so the behavior of if lang is set should be the same as if a non-existent option is set